api/src/test/java/ca/bc/gov/educ/eas/api/BaseEasAPITest.java

```diff
@@ -50,7 +50,7 @@ public SessionEntity createMockSessionEntity() {
     LocalDateTime currentDate = LocalDateTime.now();
     return SessionEntity.builder()
             .sessionID(UUID.randomUUID())
-            .courseSession(Integer.toString(currentDate.getYear()) + Integer.toString(currentDate.getMonthValue()))
+            .schoolYear(String.valueOf(currentDate.getYear()))
```

Comment:
if we aren't following the "2024/2025" syntax, are we sure the school year shouldn't be the year when the session would end (2025) rather than current year?